Common++/src/IpAddress.cpp

```diff
@@ -43,6 +46,29 @@ namespace pcpp
 	}
 
 
+	bool IPv4Address::matchSubnet(const std::string& subnet) const
+	{
+		std::stringstream ss(subnet);
+		std::string subnetOnly, subnetPrefixStr;
+		std::getline(ss, subnetOnly, '/');
+		std::getline(ss, subnetPrefixStr);
+
+		uint32_t subnetPrefix = std::stoi(subnetPrefixStr);
```

Comment:
We can also check that `subnetPrefix` is lower or equal to 32

Reply:
Done